src/source.sql

```diff
@@ -0,0 +1,86 @@
+-- Create the database 'blogeboulga' if it doesn't exist
+CREATE DATABASE IF NOT EXISTS blogeboulga;
+USE blogeboulga;
+
+-- Create the 'category' table
+CREATE TABLE category (
+                          id INT PRIMARY KEY,
+                          category_name VARCHAR(255) NOT NULL,
+                          status TINYINT(2) NOT NULL
+);
+
+-- Create the 'newsletter' table
+CREATE TABLE newsletter (
+                            id INT AUTO_INCREMENT PRIMARY KEY,
+                            email_address VARCHAR(255) NOT NULL,
+                            subscription_date DATE,
+                            unsubscribe DATE,
+                            status TINYINT NOT NULL
+);
+
+-- Create the 'user' table
+CREATE TABLE user (
+                      id INT AUTO_INCREMENT PRIMARY KEY,
+                      first_name VARCHAR(80) NOT NULL,
+                      last_name VARCHAR(80) NOT NULL,
+                      email_address VARCHAR(255) NOT NULL,
+                      login VARCHAR(80) NOT NULL,
+                      password VARCHAR(80) NOT NULL,
```

Comment:
since we gonna encrypt password, you can keep it to 255 characters

Reply:
done